src/ontology/edits/oeo-physical.omn

```diff
@@ -5710,7 +5710,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/810"
 Class: OEO_00020003
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Energy transformation is a process in which one ore more certain types of energy as input result in certain types of energy as output.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Energy transformation is a transformation in which one or more certain types of energy as input result in certain types of energy as output."@en,
         <http://purl.obolibrary.org/obo/IAO_0000233> "class added:
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/77
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/371
```

Comment:
Please update the term tracker

Reply:
Thanks for the hint. Added the term tracker again.